spa/src/pages/admin/avatar/search.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div class="grid grid-cols-1 w-full place-items-center">
+    <div class="text-center w-full text-5xl mb-1">Avatars</div>
+    <div class="grid grid-cols-2 w-4/6 justify-items-center">
+      <div>
+        Status:
+        <select v-model="status"
+                @change="getResults">
+          <option value="2">Pending Approval</option>
+          <option value="1">Live</option>
+          <option value="0">Rejected</option>
+        </select>
+      </div>
+      <div>
+        View Amount:
+        <select v-model="limit"
+                @change="getResults">
+          <option value="10">10</option>
+          <option value="20">20</option>
+          <option value="50">50</option>
+          <option value="100">100</option>
+        </select>
+      </div>
+    </div>
+    <div class="grid-cols-1 w-4/6 justify-items-center text-center">
+      Total Count: {{ totalCount }}
+    </div>
+    <table class="table-auto border-collapse">
+      <tr>
+        <th class="p-4">ID</th>
+        <th class="p-4">Name</th>
+        <th class="p-4">Member</th>
+        <th class="p-4">Access</th>
+        <th class="p-4">Status</th>
+        <th class="p-4">Files</th>
+        <th class="p-4">Actions</th>
+      </tr>
+      <tr v-for="avatar in avatars"
+          :key="avatar.id">
+        <td class="p-4">{{ avatar.id }}</td>
+        <td class="p-4">{{ avatar.name }}</td>
+        <td class="p-4">{{ avatar.username ? avatar.username : '' }}</td>
+        <td class="p-4">{{ accessLabel[avatar.private] }}</td>
+        <td class="p-4">{{ statusLabel[avatar.status] }}</td>
+        <td class="p-4">
+          <button class="btn" @click="downloadWrl(avatar.id)">WRL</button>
+          <button class="btn" @click="downloadThumbnail(avatar.id)">Thumbnail</button>
```

Comment:
Button opens up static url to image, maybe recreate overlay like object approval page?